dependencies/pom.xml

```diff
@@ -920,10 +923,10 @@
                 <artifactId>jboss-transaction-spi</artifactId>
                 <version>${version.lib.jboss.transaction-spi}</version>
                 <exclusions>
-                  <exclusion>
-                      <groupId>org.jboss.spec.javax.resource</groupId>
-                      <artifactId>jboss-connector-api_1.7_spec</artifactId>
-                  </exclusion>
+                    <exclusion>
+                        <groupId>org.jboss.spec.javax.resource</groupId>
+                        <artifactId>jboss-connector-api_1.7_spec</artifactId>
+                    </exclusion>
```

Comment:
this formating change seems weird

Reply:
Seems this is not from my PR, let's see after next push.